1209/812A/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: Danyboard M0
+owner: Danyboard
+license: GPL
+site: http://www.danyboard.com/
+source: https://github.com/Gius-8/Danyboard_M0/
+---
```

Comment:
Please adjust the title on this and the other PID to indicate which one is for the bootloader and which is for the client software.